package.json

```diff
@@ -51,7 +51,7 @@
     "tiptap-extensions": "^1.28.6",
     "underscore": "~1.9.2",
     "uniqid": "^5.2.0",
-    "v-calendar": "^1.0.1",
+    "v-calendar": "^2.3.0",
```

Comment:
Bumping a major version might be an issue for people using v-calendar themselves. This should not happen in a 3.1 release. Let's target `master`.

Reply:
Will do. Let's keep an eye open then, they've currently even have a v3 in alpha.

Reply:
I'm fine with removing `time_required` – it was never request, I think I was just overaggressively wiring up props. If you have a time field, it'll get _a_ time, and therefore be filled. Seems superfluous.